DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,26 @@ public function getConfigTreeBuilder()
                     return $v;
                 })
             ->end()
+            ->beforeNormalization() // @deprecated toolbar in 1.3.0, use profiling instead
+                ->ifTrue(function ($v) {
+                    return is_array($v) && array_key_exists('toolbar', $v) && is_array($v['toolbar']);
+                })
+                ->then(function ($v) {
+                    if (array_key_exists('enabled', $v['toolbar']) && 'auto' === $v['toolbar']['enabled']) {
+                        $v['toolbar']['enabled'] = $this->debug; // @deprecated value auto in 1.3.0
+                    }
+
+                    if (array_key_exists('profiling', $v) && is_array($v['profiling'])) {
```

Comment:
Okay, so if the new config exists: complain or simply ignore the old config?

I still don't get the default value thing: yes, defaultValue comes after the normalization, but we need to check the auto value for BC in the normalization section.